bindings/ios/MEGANode.mm

```diff
@@ -53,6 +53,14 @@ - (instancetype)clone {
     return self.megaNode ? [[MEGANode alloc] initWithMegaNode:self.megaNode->copy() cMemoryOwn:YES] : nil;
 }
 
+- (BOOL)isEqual:(id)object {
```

Comment:
We must override `hash` method to make sure the hash values are same for 2 equal objects. Otherwise, a hash based collection will malfunction.

Reply:
Please checkout this link, it could be helpful to define your hash code.
https://nshipster.com/equality/#hashing

Reply:
Right, Thanks. I have added the hash method.